tests/certification/state/azure/blobstorage/README.md

```diff
@@ -0,0 +1,17 @@
+# Azure Blob storage certification testing
+
+This project aims to test the Azure Blob storage State Store component under various conditions.
+
+## Test plan
+
+## Basic Test for existing container for CRUD operations:
+1. Able to create and test connection.
+2. Able to do set, fetch, delete.
```

Comment:
We really do not need to verify these things as this is all handled by conformance test. This just leads to code duplication.

All that needs to be tested is that we can authenticate and do anything.